Document using custom settings with clickhouse and chart (#2833)
PostHog/posthog.com: contents/docs/self-host/deploy/configuration.md

To use an external `ClickHouse` cluster, set `clickhouse.enabled` to `false` and set `clickhouse.host`, `clickhouse.database`, `clickhouse.user` and `clickhouse.password`.

#### Custom settings

It's possible to pass custom settings to ClickHouse. This might be needed to e.g. set query time limits or increase max memory usable by clickhouse.

> To do so, change `clickhouse.profiles` setting as below. The `default` profile is used by posthog for all queries.

Reviewer: ```suggestion
To do so, you can override the `clickhouse.profiles` values as below. The `default` profile is used by PostHog for all queries.
```
